var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -332,57 +165,13 @@ class Warpx(CMakePackage, PythonExtension):
         depends_on("py-numpy@1.15.0:", type=("build", "run"))
         depends_on("py-mpi4py@2.1.0:", type=("build", "run"), when="+mpi")
         depends_on("py-periodictable@1.5:1", type=("build", "run"))
-        depends_on("py-picmistandard@0.28.0", type=("build", "run"), when="@23.11:24.07")
-        depends_on("py-picmistandard@0.29.0", type=("build", "run"), when="@24.08")
         depends_on("py-picmistandard@0.30.0", type=("build", "run"), when="@24.09:")
```

Comment:
Should this be
```suggestion
        depends_on("py-picmistandard@0.30.0", type=("build", "run"))
```
since the minimum version is now `24.10` anyways (if I'm not mistaken)?

Reply:
Yes, I kept this so we do not forget to update and constrain it once we add the next release. We will have to change this line then anyway.